manager/config/prod/deployment_configmap.yaml

```diff
@@ -6,7 +6,7 @@ kind: ConfigMap
 metadata:
   name: m4d-config
 data:
-  SECRET_PROVIDER_URL: "http://secret-provider.m4d-system.svc.cluster.local:5555/get-secret"
+  SECRET_PROVIDER_URL: "http://secret-provider:5555/get-secret"
```

Comment:
This won't work, it is passed to modules that run in different namespace

Reply:
I tested it on Kind. The suffixes  in /etc/resolve are: m4d-system.svc.cluster.local svc.cluster.local cluster.local
That's why I omitted the m4d-system

Reply:
I validated and the arrow module did not fail... maybe the address is cached somehow? is there a way to look at the logs? thanks

Reply:
I've added the m4d-system to secret-provider. Thanks